docs/state.md

```diff
@@ -21,10 +21,9 @@ AppStore
   },
   about: {
     brave: {
-      versionInformation: [{
-        name: string,
-        version: string
-      }] // used on about:brave. not persisted (removed on save)
+      versionInformation: {
+        [name]: string
+      } // map of property name to version. used on about:brave. not persisted (removed on save)
```

Comment:
I checked my prod user profile and versionInformation is present.

cat ~/Library/Application\ Support/brave/session-store-1 | python -mjson.tool | less